apps/desktop/src/components/StackTabNew.svelte

```diff
@@ -13,10 +13,10 @@
 
 	type Props = {
 		projectId: string;
-		selectedId: string | undefined;
+		tabOverflow?: boolean;
 	};
 
-	const { projectId }: Props = $props();
+	const { projectId, tabOverflow = false }: Props = $props();
```

Comment:
It feels a bit like stuttering when we repeat part of the name of the component. Can we call this prop `overflow`, like the css class it controls?

Reply:
Merged :+1: